comms/docker-compose.yml

```diff
@@ -156,3 +156,19 @@ services:
         condition: service_healthy
       nats-discovery:
         condition: service_healthy
+
+  # SSH server for sshuttle
+  # we use separate container instead of host machine for dns/auto-hosts functionality
+
+  sshuttle-server:
+    image: linuxserver/openssh-server:latest # don't pin
+    command: bash /tmp/dev-tools/startup/sshuttle-server.sh
+    environment:
+      PUBLIC_KEY_FILE: "/tmp/dev-tools/keys/dev.pub"
+      USER_NAME: "sshuttle"
+    ports:
+      - "2223:2222"
+    volumes:
+      - ../dev-tools/:/tmp/dev-tools
+    deploy:
+      mode: global
```

Comment:
I'm confused why we need this when you already added that comms-ingress.sh script? if we do need it tho is there any way to use the same sshuttle containers as the rest of audius-compose? it's nice to test some things isolated with just a minimal comms setup that uses [this line](https://github.com/AudiusProject/audius-protocol/blob/main/comms/Makefile#L4) for ingress to an nginx container that routes to the correct container based on hostname